api/controllers/health.go

```diff
@@ -0,0 +1,33 @@
+package controllers
+
+import (
+	"github.com/quintilesims/layer0/common/config"
+	"github.com/zpatrick/fireball"
+)
+
+type HealthController struct {
+	Config  config.APIConfig
```

Comment:
why does it need either of these?

Reply:
:thinking: I suppose given that the API doesn't have any sort of cli flag or environment variable that corresponds with this function, we wouldn't need to have this. If the API changes in some way that would add a cli flag or env var for this controller, then I would suppose we would need these.

I'll remove them.